terraform/clickstream_analytics/variables.tf

```diff
@@ -0,0 +1,52 @@
+#  Copyright 2024 Google LLC
```

Comment:
You can probably adapt these variables to your specific pipeline (e.g. Bigtable zone location).